modules/ai/assets/js/editor/pages/get-started/index.js

```diff
@@ -16,15 +16,15 @@ const GetStarted = ( { onSuccess } ) => {
 		<Stack alignItems="center" gap={ 4 }>
 			<AIIcon sx={ { color: 'text.primary', fontSize: '60px', mb: 3 } } />
 
-			<Typography variant="h4">{ __( 'Step into the future with Elementor AI', 'elementor' ) }</Typography>
+			<Typography variant="h4" sx={ { color: 'text.primary' } }>{ __( 'Step into the future with Elementor AI', 'elementor' ) }</Typography>
 
 			<Typography variant="body2">{ __( 'Create smarter with AI text and code generators built right into the editor.', 'elementor' ) }</Typography>
 
 			<Stack direction="row" gap={ 4 } alignItems="flex-start">
 				<Checkbox id="e-ai-terms-approval" color="secondary" sx={ { p: 0 } } onChange={ () => setIsTermsChecked( ( prevState ) => ! prevState ) } />
 
 				<Stack>
-					<Typography variant="caption" sx={ { maxWidth: 520 } } component="label" for="e-ai-terms-approval">
+					<Typography variant="caption" sx={ { maxWidth: 520 } } component="label" htmlFor="e-ai-terms-approval">
 						{ __( 'I approve the ', 'elementor' ) }
 						<Link href="https://go.elementor.com/ai-terms/" target="_blank" color="info.main">{ __( 'Terms of Service', 'elementor' ) }</Link>
 						{ ' & ' }
```

Comment:
Try using this for the translations:

https://make.wordpress.org/core/2020/07/17/introducing-createinterpolateelement/

The problem is that it'll probably add a bunch of JS into our code, since it's not coming from the i18n package but from the react wrapper thing. Not sure how to handle that... 
@Nevoss @KingYes?

Reply:
After consulting with @Nevoss it might be problematic due to the reason you've mentioned + the fact that it's available only on WP 5.5 - we don't have a better option for now (we need to think about what to do with it later on).